pkg/agent/openflow/client.go

```diff
@@ -158,11 +152,10 @@ type Client interface {
 	// SNAT with the Openflow NAT action.
 	InstallExternalFlows(exceptCIDRs []net.IPNet) error
 
-	// InstallSNATMarkFlows installs flows for a local SNAT IP. On Linux, a
+	// InstallSNATMarkFlow installs flow for a local SNAT IP. On Linux, a
 	// single flow is added to mark the packets tunnelled from remote Nodes
-	// that should be SNAT'd with the SNAT IP. On Windows, an extra flow is
-	// added to perform SNAT for the marked packets with the SNAT IP.
-	InstallSNATMarkFlows(snatIP net.IP, mark uint32) error
+	// that should be SNAT'd with the SNAT IP.
+	InstallSNATMarkFlow(snatIP net.IP, mark uint32) error
 
 	// UninstallSNATMarkFlows removes the flows installed to set the packet
```

Comment:
If you rename InstallSNATMarkFlows(), you should rename this one as well? (I feel no need to rename either, but up to you).

Reply:
I changed it back.